tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -25,22 +25,27 @@ sub get_script_run {
     my $max_test_time        = get_var("MAX_TEST_TIME", "36000");
 
     handle_sp_in_settings_with_sp0("PRODUCT_UPGRADE");
-    my $product_upgrade = get_var("PRODUCT_UPGRADE", "sles-12-sp2-64");
+    my $product_upgrade = get_var("PRODUCT_UPGRADE");
 
     handle_sp_in_settings_with_fcs("GUEST_LIST");
-    my $guest_list = get_var("GUEST_LIST", "sles-12-sp1-64");
+    my $guest_list = get_var("GUEST_LIST");
+    $guest_list =~ s/sles-15-sp0/sles-15-fcs/i;
 
     $pre_test_cmd = "$pre_test_cmd -p $product_upgrade -r $product_upgrade_repo -t $max_test_time -g \"$guest_list\"";
+    if (get_var("SKIP_GUEST_INSTALL") && check_var('ARCH', 'x86_64')) {
+        $pre_test_cmd .= " -k";
+    }
     return $pre_test_cmd;
 }
 
 sub analyzeResult {
     my ($self, $text) = @_;
     my $result;
     # In the case the log does not tell the exact failure
-    if ($text !~ /Overall guest upgrade result is:(.*)Test done/s) {
-        $result->{'subtest'}->{status} = 'FAILED';
-        $result->{'subtest'}->{error}  = 'Please check the guest_upgrade_test log in the guest_upgrade_run-guest-upgrade-logs.tar.gz';
+    if (($text !~ /Overall guest upgrade result is:(.*)Test done/s) && ($text !~ /Guest Restoration Error:/s)) {
+        my $subtest = get_var('GUEST_LIST');
```

Comment:
this may represent 2 guests on xen

Reply:
Yes. In this case, no log indicate which guest or both guests have failures, or what the failures are. so just put the guest setting here. When more such cases happen, we can refine it.